html/index.html

```diff
@@ -1,26 +1,27 @@
 <!DOCTYPE html>
 <head>
     <title>My Personal Website</title>
-    <style></style>
 </head>
 <body>
     <header>
-        <img src="/media/cc0-images/grapefruit-slice-332-332.jpg">
+        <img alt="Grapefruit slice atop a pile of other slices" src="/media/cc0-images/grapefruit-slice-332-332.jpg">
         <nav class="quicknav">
             <a href="#News">News</a>
```

Comment:
You need to adjust `#News` as below you wrote `id="news"`. :-) So `#news` it is.